data_loader/training_storer.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from data_loader.spreadsheet_dao import *
+
```

Comment:
training_data_service, training_service would be the more conventional name (agreed in the industry)

so... can have training_service that uses spread_sheet_dao AND in 6 month we can make it uses some futuristic postgres_dao...

BUTTTT!
[this part is not a must, but bare with me if it trigger your curiosity!]
for this to happen we need to use interfaces, and classes

can you think of a way to design this?

Reply:
well my curiosity is triggered but I would like some more elaborate explanation about this.
not sure about the interfaces part yet.